### ISSUE TITLE
Build fails, cannot fetch

### ISSUE THREAD

**Issue by eserte** [sanko/alien-fltk2#1]
Trying to build Alien-FLTK2 currently always fails, apparently because it's not possible to fetch the original. It always ends with something like

```
[13/16] Trying HTTP mirror based in California, USA... done.
Checking MD5 hash of archive... Can't use an undefined value as a HASH reference at inc/MBX/Alien/FLTK/Base.pm line 1019.
```

**Comment by sanko** [comment]
Yep. The 2.x branch of fltk was abandoned upstream and snapshots are no longer available for download. [Edit: Removing dist from PAUSE... when I get to my computer...]

This also means the FLTK module on CPAN is broken until I rewrite it for the older (but, ironically, still being developed) 1.3.x branch. Not an easy task since the API between the toolkits is 100% different.
